.circleci/config.yml

```diff
@@ -137,9 +137,7 @@ jobs:
             KOFLAGS: --jobs=4
             DIST_DIR: /tmp/dist/
           command: |
-            IMAGE_TAG=${CIRCLE_TAG:-${CIRCLE_SHA1}} \
-            KOFLAGS="--platform linux/amd64,linux/arm64" \
```

Comment:
Actually your change overrode the `--jobs=4` KOFLAG. Please move the new args to `KOFLAGS` to the env variable on line 137 for example.

Reply:
Nice catch, thanks